### DISCUSSION TITLE
目录重命名后，搜索还是能搜到重命名前的目录下的文件

### DISCUSSION THREAD

**Discussion by zmy0516** [AlistGo/alist#1037] [Q&A]
比如A目录下有文件B，现在把A目录名改成AA，当我搜索文件B的时候，会出现两条记录，一条A/B，一条AA/B

**Comment by anwen-anyi** [comment]
缓存的问题吧，这个数据是从数据库返回来的~数据库的数据还没清掉的原因吧（**个人见解不代表程序开发者的回答**）

**Reply by zmy0516** [reply]
我看了下是存db数据库里面，我把表清了，重建，应该是可以了。但是还是希望可以自动的清除这些过期数据

**Reply by anwen-anyi** [reply]
> 我看了下是存db数据库里面，我把表清了，重建，应该是可以了。但是还是希望可以自动的清除这些过期数据

60分钟后好像会自动清除...就是刷新缓存的时候（我记得是这样的..）

**Reply by zmy0516** [reply]
好几天都没刷新

**Comment by xhofe** [comment]
**Experimental** function, not recommended as it's still under development
